src/main/kotlin/minesweeper/MineSweeperMain.kt

```diff
@@ -14,5 +15,31 @@ fun main() {
 
     val mineMap = MineMap(width, height)
     mineMap.makeMine(mineCount)
-    OutputView.showGameResult(mineMap)
+    OutputView.showGameStartMessage()
+
+    inputOpenPosition(mineMap)
+}
+
+fun inputOpenPosition(mineMap : MineMap) {
+    val minePosition = InputView.inputOpenPosition()
+    openTile(minePosition, mineMap)
+}
+
+private fun openTile(position: MinePosition, mineMap : MineMap) {
+    mineMap.openTile(position) {
+        isLoose ->
```

Comment:
스코프 함수 사용 시 대개 아래처럼 사용하시는 편입니다. 아래처럼 사용하시는건 어떨까요?

```suggestion
    mineMap.openTile(position) { isLoose ->
```

Reply:
말씀하신 느낌이 일반적이라는 말씀이신거죠? 수정해보겠습니다.